File: app/src/main/java/eu/qm/fiszki/activity/TutorialActivity.java
package eu.qm.fiszki.activity;

/**
 * Created by bgood on 2016-04-14.
 */

import android.content.Intent;
import android.os.Bundle;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.view.KeyEvent;
import android.view.View;

import eu.qm.fiszki.R;
import eu.qm.fiszki.tutorial.ColorPage;

public class TutorialActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getSupportActionBar().hide();
        setContentView(R.layout.activity_tutorial);

        if (savedInstanceState == null) {
            replaceTutorialFragment();
        }
    }

    @Override
    public boolean onKeyDown(int keyCode, KeyEvent event) {
        if (keyCode == KeyEvent.KEYCODE_BACK) {
            Intent myIntent = new Intent(this, MainActivity.class);
            startActivity(myIntent);
            finish();
        }
        return super.onKeyDown(keyCode, event);
    }

    public void replaceTutorialFragment() {
        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.replace(R.id.container, new ColorPage());
        fragmentTransaction.commit();
    }

    public void goButton(View view) {
        Intent myIntent = new Intent(this, MainActivity.class);
        startActivity(myIntent);
        finish();
    }
}

File: app/src/main/java/eu/qm/fiszki/tutorial/TutorialActivity.kt
package eu.qm.fiszki.tutorial

import android.content.Intent
import android.os.Bundle
import android.view.KeyEvent
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import eu.qm.fiszki.NightModeController
import eu.qm.fiszki.R
import eu.qm.fiszki.activity.MainActivity
import eu.qm.fiszki.tutorial.page.ColorPage

class TutorialActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        NightModeController(this).useTheme()
        setContentView(R.layout.activity_tutorial)

        if (savedInstanceState == null) {
            replaceTutorialFragment()
        }
    }

    override fun onKeyDown(keyCode: Int, event: KeyEvent): Boolean {
        if (keyCode == KeyEvent.KEYCODE_BACK) {
            startActivity(Intent(this, MainActivity::class.java))
            finish()
        }
        return super.onKeyDown(keyCode, event)
    }

    fun replaceTutorialFragment() {
        supportFragmentManager.beginTransaction().apply {
            replace(R.id.container, ColorPage())
            commit()
        }
    }

    fun goButton(view: View) {
        startActivity(Intent(this, MainActivity::class.java))
        finish()
    }

    fun skipButton(view: View) {
        startActivity(Intent(this, MainActivity::class.java))
        finish()
    }
}
